src/Raven.Server/ServerWide/ClusterStateMachine.cs

```diff
@@ -2696,6 +2700,10 @@ private void SqueezeDatabasesToSingleNodeCluster(ClusterOperationContext context
             {
                 using (var rawRecord = ReadRawDatabaseRecord(context, name))
                 {
+                    // we can't squeeze sharded database to a single node
+                    if (rawRecord.IsSharded())
+                        continue;
+
```

Comment:
huh? That is when we have a single node in the cluster, no?

Reply:
that is when we do a hard reset to a single node

Reply:
So why aren't we handling this for shards a well?

Reply:
for non sharded we will end up with 1 node database or with no database at all.
here we might end up only with some of the shards... let's we will end up only with $1,$4 out of 5 shards.

Reply:
Oh, I see. Make sense, then

Reply:
Maybe add alert?